.circleci/config.yml

```diff
@@ -79,7 +73,7 @@ jobs:
       - run:
           name: Fetch binaries
           command: |
-            curl -fsSL -o /workspace/bin/linuxkit https://152-46932243-gh.circle-artifacts.com/0/linuxkit-linux-amd64
+            curl -fsSL -o /workspace/bin/linuxkit https://183-46932243-gh.circle-artifacts.com/0/linuxkit-linux-amd64
```

Comment:
since https://github.com/linuxkit/linuxkit/pull/2761 just got merged do you want to roll that in here and ditch `moby`

Reply:
I guess I should, yes. After I figured out where I went wrong with my yaml-fu for the existing stuff ;-)